framework/csp_test.py

```diff
@@ -52,15 +52,15 @@ def test_get_default_policy__strict(self):
     """We can get the regular strict policy."""
     policy = csp.get_default_policy(nonce='12345')
     self.assertCountEqual(list(csp.DEFAULT_POLICY.keys()), list(policy.keys()))
-    self.assertIn('strict-dynamic', policy['script-src'])
+    self.assertIn('\'strict-dynamic\'', policy['script-src'])
     self.assertIn("'nonce-12345'", policy['script-src'])
 
   @mock.patch('framework.csp.USE_NONCE_ONLY_POLICY', True)
-  def test_get_default_policy__strict(self):
+  def test_get_default_policy__strict_two(self):
     """We can get the even stricter nonce-only policy."""
     policy = csp.get_default_policy(nonce='12345')
     self.assertCountEqual(list(csp.NONCE_ONLY_POLICY.keys()), list(policy.keys()))
-    self.assertNotIn('strict-dynamic', policy['script-src'])
+    self.assertNotIn('\'strict-dynamic\'', policy['script-src'])
```

Comment:
To be honest, I did this to both of these test functions because the first one was failing with this error:
```
AssertionError: 'strict-dynamic' not found in ["'strict-dynamic'", "'unsafe-inline'", 'https:', 'http:', 'https://www.gstatic.com', "'nonce-12345'"]
```

The first function was technically not running because it was being overwritten due to having the same name as the second function. If it is better to not have these quotes , we will need to change the `DEFAULT_POLICY` constant that has extra quotes defined here:
https://github.com/GoogleChrome/chromium-dashboard/blob/ad469ade08eb7abc79896736e1073fa0fd8cbe5c/framework/csp.py#L36-L52

(I am not familiar with this so I am fine with any fix that makes this work as intended 😅 )

Reply:
Oh, I see.  It is not testing for a substring, it is testing for an item in a list.  That item needs to be an exact match, so it needs the quote characters.  This looks right.